website-custom-views/src/content/development/testing.mdx

```diff
@@ -235,3 +235,84 @@ const renderView = (options = {}) => {
   });
 };
 ```
+
+# End-to-End
```

Comment:
Since we're adding docs here, maybe we should include @Anshuman71 as a reviewer?